image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
@tonistiigi Well, I was surprised to find this on "export", as that means something different, as well.

Shouldn't these be separate concepts?